src/content/en/tools/workbox/guides/migrations/migrate-from-v2.md

```diff
@@ -166,6 +166,29 @@ const cdnTransform = (manifestEntries) => {
 };
 ```
 
+- The `getFileManifestEntries()` function has been renamed to `getManifest()`, and the promise
+returned now [includes additional
+information](/web/tools/workbox/reference-docs/latest/module-workbox-build#.getManifest) about the
```

Comment:
The link for this doc doesn't work.  Is that content coming on https://developers.google.com/web/tools/workbox/reference-docs/latest/module-workbox-build?

Reply:
Yup—as soon as https://github.com/google/WebFundamentals/pull/5907 is deployed to the production website.